cluster-autoscaler/cloudprovider/azure/azure_cloud_provider_test.go

```diff
@@ -56,7 +56,7 @@ func newTestAzureManager(t *testing.T) *AzureManager {
 			VMType:              vmTypeVMSS,
 			MaxDeploymentsCount: 2,
 			Deployment:          "deployment",
-			EnableForceDelete:   true,
+			EnableForceDelete:   false,
```

Comment:
why false now by default?

Reply:
Some unit tests were not synced perfectly apparently. Fixed.